views/account/login/index.ejs

```diff
@@ -6,49 +6,92 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Log In</title>
 
+    <!-- 
+    When index.ejs is rendered, 2 requests will be 
+    sent to the server for the CSS and JS files. 
+    -->
+    <link rel="stylesheet" href="/account/login/style" />
+    <script src="/account/login/main" type="module"></script>
+
     <!-- Bootstrap links to make it work  -->
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
-    <link href="http://netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.css" rel="stylesheet" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css"
+      rel="stylesheet"
```

Comment:
I see what you're doing here and I think it's a good idea but it makes me wonder if we should have some sort of prettier module to standardize it across the app

I could also just do a better job of formatting lol.

Reply:
Ah yes, I completely forgot about Prettier. Any file I touch gets changed around, so don't feel like I'm targeting your style haha.

Reply:
It's always personal. Nah I get it, so it might be good to bring it up to the rest of the team if everyone has a different formatting style